packages/merge-styles/src/IStyleSet.ts

```diff
@@ -5,18 +5,13 @@ export type Diff<T extends keyof any, U extends keyof any> = ({ [P in T]: P } &
 
 export type Omit<U, K extends keyof U> = Pick<U, Diff<keyof U, K>>;
 
-export type __ReduceToFunction<T> = T extends (...args: any[]) => any ? T : never;
-
 /**
  * Helper function whose role is supposed to express that regardless if T is a style object or style function,
  * it will always map to a style function.
  *
- * Note: the commmented segment is the right expression but it is buggy in 2.8.2. Once Fabric upgrades to Typescript 3,
- *  we should uncomment the commented segmment below as well as uncomment the test in mergeStyleSets.test.ts.
- *
- * See https://github.com/OfficeDev/office-ui-fabric-react/issues/6124.
+ * @author [Nimelrian](https://github.com/Nimelrian) in https://github.com/OfficeDev/office-ui-fabric-react/issues/6615
```

Comment:
could you remove the @author tag here? I don't think there's value in doing that. We collectively own the code.

Reply:
@dzearing  That's just to give credit to @Nimelrian. If I did not do this, it would look like I devised this clever way that works in TS 2.8 as well (the way I came up with is buggy in 2.8 that they fixed in subsequent Typescript releases.)

LMK if you still think I should remove it.

@Nimelrian feel free to chime in if you lean one way or the other on this issue too.

Reply:
Just to be clear, this is not to say that @Nimelrian owns the code and he will be responsible for fixing all bugs - this is just proper attribution that this is a valued community contribution.

Reply:
You're free to remove that author tag., I have no feelings either way about it. Open Source is a community effort. A mention in the release notes would be enough :)